source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -188,17 +204,25 @@ TEST_F(NiDAQmxDriverApiTests, CreateDOChannel_Succeeds)
   EXPECT_SUCCESS(status, response);
 }
 
-TEST_F(NiDAQmxDriverApiTests, AIVoltageChannel_StartTaskStopTask_Succeeds)
+TEST_F(NiDAQmxDriverApiTests, AIVoltageChannel_ReadAIData_ReturnsDataInExpectedRange)
 {
+  const double AI_MIN = 1.0;
+  const double AI_MAX = 10.0;
   CreateAIVoltageChanResponse create_channel_response;
-  create_ai_voltage_chan(create_channel_response);
+  create_ai_voltage_chan(AI_MIN, AI_MAX, create_channel_response);
 
   StartTaskResponse start_response;
   auto start_status = start_task(start_response);
+  ReadAnalogF64Response read_response;
+  auto read_status = read_analog_f64(100, 100, read_response);
   StopTaskResponse stop_response;
   auto stop_status = stop_task(stop_response);
 
+  EXPECT_EQ(read_response.read_array_size(), 100);
+  EXPECT_THAT(read_response.read_array(), Each(Not(Lt(AI_MIN))));
```

Comment:
[FYI] This is just in case you didn't see these matchers, but you could use >= and <= instead if you wanted for `Each(Ge(AI_MIN))` and `Each(Le(AI_MAX))`. But I prefer the way you have it, it reads better.